### DISCUSSION TITLE
Planned release date v2.7/v3

### DISCUSSION THREAD

**Discussion by divine** [api-platform/core#4729] [Q&A] [answered]
Hello,

Is there any planned release date for v2.7/v3?

Thanks!

**Comment by darthf1** [comment]
See https://github.com/api-platform/api-platform/discussions/2020

**Comment by soyuka** [comment, marked as answer]
We would like to release a 2.7 beta within the next week.

For the 3.0 we'll start the work on it this month (which means removing deprecations and cleaning code). The 3.0 will require php >= 8.1. We can't say for sure how long it'll take but I'd like for it to be released end June.

**Reply by divine** [reply]
Thank you for your work @soyuka! Is there anything that I could help with?

Thanks!
